packages/host/app/resources/room.ts

```diff
@@ -83,28 +83,12 @@ export class RoomResource extends Resource<Args> {
   @service declare private cardService: CardService;
 
   modify(_positional: never[], named: Args['named']) {
-    if (named.roomId) {
-      if (this.isNewRoom(named.roomId)) {
-        this.resetCache();
-      }
```

Comment:
this is not a good strategy. it defeats the purpose of the cache and its a really awkward way to constrain the values returned from this reource to only those that pertain to this room. A better approach is to keep this cache, but then filter out only items that pertain to this room in our various getters, and to make sure those getters are cached.